recipe/meta.yaml

```
@@ -46,4 +46,5 @@ about:
 
 extra:
   recipe-maintainers:
+    - consideratio
     - bollwyvl
```

```suggestion
    - bollwyvl
    - consideratio
    - minrk
```

I'm adding @minrk as well.